x/nft/username/codec.proto

```diff
@@ -0,0 +1,23 @@
+syntax = "proto3";
+
+package username;
+
+import "github.com/iov-one/weave/x/nft/codec.proto";
+import "github.com/gogo/protobuf/gogoproto/gogo.proto";
+
+
+message UsernameToken {
```

Comment:
We are assuming that the username is stored within the id, right?
Makes sense, as it should be unique in this case.

Reply:
yes, Id = name